apps/admin-panel/src/App.jsx

```diff
@@ -1655,6 +1656,58 @@ const TpCompanyProfileListFilters = (props) => (
   </Filter>
 )
 
+const ConditionalTpCompanyProfileHowDidHearAboutRediKeyFieldShow = (props) => {
+  return props.record?.howDidHearAboutRediKey ? (
```

Comment:
I guess this conditional is superfluous since we want to show this field in any case?